plugins/module_utils/dnac.py

```diff
@@ -65,9 +65,11 @@ def __init__(self, module):
                                         }
         self.dnac_log = dnac_params.get("dnac_log")
 
-        # Check if 'dnac_log_level' in the playbook params. If available,
-        # convert it to uppercase; otherwise, set it to 'INFO'
-        self.dnac_log_level = dnac_params.get("dnac_log_level", "INFO").upper()
+        self.dnac_log_level = dnac_params.get("dnac_log_level")
+        self.is_valid_log_level()
```

Comment:
It may again throw an exception. If we don't have log_level in playbook or in any of our modules, then we check valid_log_level and this may throw an exception. can you please check it?

Reply:
Do you would like to put this check after the below one?
```
       if self.dnac_log_level is None:
            self.dnac_log_level = "INFO"
```